### ISSUE TITLE
Rework ashen passage

### ISSUE THREAD

**Issue by PowerfulBacon** [BeeStation/BeeStation-Hornet#11533]
I want it so that instead of functioning as a jaunt, ashen passage gives you a set amount of time where you can move through doors, tables and other partially passable materials.
There should be some kind of decent animation when this happens, and you should phase through immediately to the other side.
You should not be able to pass through solid tiles that you wouldn't normally be able to pass like windows or walls.
Perhaps an item should be made to activate this ability, instead of a spell with an easy crafting recipe such as organic matter + a catalyst lighter.

**Comment by Tsar-Salat** [comment]
Yeah, I agree. The ashen jaunt right now is just not fun to deal with. 
When you are in a chase with the heretic and they just jaunt away for the 10th time, you start to lose your shit, and consider getting acid to throw on them.

Regarding the 'activator' were you thinking kinda like Pyro from x-men?

**Comment by PowerfulBacon** [comment]
I was just thinking an item that you use in your hand to trigger the ashen jaunt, or one which grants an action to activate it.
